test/backend/NMLUnitTestSuite.scala

```diff
@@ -0,0 +1,144 @@
+package backend
+
+import java.io.ByteArrayInputStream
+
+import com.scalableminds.webknossos.datastore.SkeletonTracing._
+import com.scalableminds.webknossos.datastore.VolumeTracing.VolumeTracing
+import com.scalableminds.webknossos.datastore.geometry.{Point3D, Vector3D}
+import com.typesafe.scalalogging.LazyLogging
+import models.annotation.nml.NmlParser
+import net.liftweb.common.{Box, Full}
+import org.scalatest.FlatSpec
+import play.api.libs.iteratee.Iteratee
+import reactivemongo.bson.BSONObjectID
+import utils.ObjectId
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+
+class NMLUnitTestSuite extends FlatSpec with LazyLogging {
+  logger.debug(s"test run")
+  val timestamp = 123456789
+
+  def getObjectId = ObjectId.fromBsonId(BSONObjectID.generate)
+
+  def getParsedTracing(tracing: SkeletonTracing) = {
```

Comment:
I would rather rename this to something more active, as it is not a typical `getter`. Maybe `writeAndParseTracing`.